polyfill/test/calendar.mjs

```diff
@@ -150,6 +150,32 @@ describe('Calendar', () => {
     });
   });
 });
+describe('Built-in calendars (not standardized yet)', () => {
+  describe('gregory', () => {
+    it('era AD', () => {
+      const date = Temporal.Date.from('1999-12-31[c=gregory]');
+      equal(date.era, 'ad');
+      equal(date.year, 1999);
+    });
+    it('era BC', () => {
+      const date = Temporal.Date.from('-000001-12-31[c=gregory]');
+      equal(date.era, 'bc');
```

Comment:
is lowercase "ad" and "bc" a convention in 402?  If not, seems unexpected because I've only seen those capitalized like "BC" or "AD"

Reply:
These are string IDs - they aren't intended to be human-readable, I think of `era: 'bc'` more like `roundingMode: 'trunc'`. When printed with `toLocaleString()` in an English locale, they'd be printed as "BC" or "B.C" or "BCE" or whatever.

These string IDs are just for demonstration anyway - their exact values would have to be specified by ECMA-402, because the `gregory` calendar is in that domain.